samples/app/MeshAssimp.cpp

```diff
@@ -230,6 +503,22 @@ bool MeshAssimp::setFromFile(const Path& file,
             // we only support triangles
             aiProcess_Triangulate);
 
+    scene = importer.ApplyPostProcessing(aiProcess_CalcTangentSpace);
+    size_t index = importer.GetImporterIndex(file.getExtension().c_str());
+    const aiImporterDesc* importerDesc = importer.GetImporterInfo(index);
+    bool isGLTF = false;
+    if (!strncmp("glTF2 Importer", importerDesc->mName, 14)) {
```

Comment:
Ah, ok, this detection scheme makes sense.  Could be a one-liner :)

Reply:
Errrr indeed. Booleans are hard :)